### ISSUE TITLE
Android: Drawer

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30875]
## Description
Drawer cannot be opened with Talkback running, but once opened it is focusable and has the proper "dismiss" action.

## React Native version:
v0.63

**Comment by blavalla** [comment]
Drawer opening actually works properly if there is a button to trigger it, rather than relying solely on the swipe gestures.  I think this is   totally fine, as the drawer pattern used on Android tends to have a trigger element anyways, since hidden edge swipes are so undiscoverable.

Closing out.
